sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/README.md

```diff
@@ -1,36 +1,145 @@
 # Azure WebJobs Event Hubs client library for .NET
 
-TODO
+This extension provides functionality for accessing Azure Event Hubs in Azure Functions.
 
 ## Getting started
 
 ### Install the package
 
-Install the Event Grid extension with [NuGet][nuget]:
+Install the Event Hubs extension with [NuGet](https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.EventHubs):
 
 ```Powershell
 dotnet add package Microsoft.Azure.WebJobs.Extensions.EventHubs --version 5.0.0-beta.1
 ```
 
 ### Prerequisites
 
-TODO
+- **Azure Subscription:**  To use Azure services, including Azure Event Hubs, you'll need a subscription.  If you do not have an existing Azure account, you may sign up for a [free trial](https://azure.microsoft.com/free) or use your [Visual Studio Subscription](https://visualstudio.microsoft.com/subscriptions/) benefits when you [create an account](https://account.windowsazure.com/Home/Index).
+
```

Comment:
You may want to include the Azure storage account information here as well.   If you're so inclined, you can steal verbiage from the [Processor README](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/eventhub/Azure.Messaging.EventHubs.Processor).